packages/yarnpkg-core/sources/scriptUtils.ts

```diff
@@ -66,15 +66,13 @@ export async function detectPackageManager(location: PortablePath): Promise<Pack
         case `yarn`: {
           const packageManager = Number(major) === 1 ? PackageManager.Yarn1 : PackageManager.Yarn2;
           return {packageManagerField: true, packageManager, reason};
-        } break;
-
+        }
         case `npm`: {
           return {packageManagerField: true, packageManager: PackageManager.Npm, reason};
-        } break;
-
+        }
         case `pnpm`: {
           return {packageManagerField: true, packageManager: PackageManager.Pnpm, reason};
-        } break;
```

Comment:
I'm not sure why, but my IDE constantly removes those statements by itself. It's technically correct, but really annoying to have those unrelated changes every time I update the file.

Reply:
Assuming you're using VSCode, global eslint config or a different extension perhaps?